api/auth/jwtUtils.js

```diff
@@ -9,57 +9,60 @@ const HS256 = 'HS256';
 /**
  * Returns a signed JWT with content as the payload. Inverse function for
  *   verifyWebToken(token).
+ * @name signWebToken
  * @param {Object} payload - payload of information to be stored within the JWT
  * @returns {String} signed JWT
  */
-const signWebToken = payload => {
-  const options = {
-    algorithm: HS256,
-    expiresIn: `${SESSION_LIFETIME_MINUTES || 5}m`,
-    issuer: TOKEN_ISSUER
-  };
+const signWebTokenOptions = {
+  algorithm: HS256,
+  expiresIn: `${SESSION_LIFETIME_MINUTES || 5}m`,
+  issuer: TOKEN_ISSUER
+};
 
-  return jwt.sign(payload, SESSION_SECRET, options);
+const signWebToken = payload => {
+  return jwt.sign(payload, SESSION_SECRET, signWebTokenOptions);
 };
 
 /**
  * Returns the payload from the signed JWT, or false. Inverse function for
  *   signWebToken(payload).
+ * @name verifyWebToken
  * @param {String} token - signed JWT
  * @returns {(Object|Boolean)} JWT payload, or false
  */
-const verifyWebToken = token => {
-  const options = {
-    algorithms: [HS256],
-    issuer: TOKEN_ISSUER
-  };
+const verifyWebTokenOptions = {
+  algorithms: [HS256],
+  issuer: TOKEN_ISSUER
+};
 
+const verifyWebToken = token => {
   let payload;
-
   try {
-    payload = jwt.verify(token, SESSION_SECRET, options);
+    payload = jwt.verify(token, SESSION_SECRET, verifyWebTokenOptions);
   } catch (err) {
     logger.error(token, `invalid token: ${err.message}`);
     return false;
   }
-
   return payload;
 };
 
 /**
  * Extracts the JWT from the Request Authorization Header.
+ * @name jwtExtractor
  * @param {Object} req - request
  * @returns {(String|null)} JWT string or null
  */
 const jwtExtractor = req => {
-  let token = req.get('Authorization');
-  if (!token) return null;
-  token = token.replace('Bearer ', '');
-  return token;
+  const token = req.get('Authorization');
+  if (!token || !token.toLowerCase().match(/^bearer\s.+\..+\..+/)) return null;
+  const [temp, result] = token.split(' ');  // eslint-disable-line no-unused-vars
```

Comment:
As a future note, one convention for throwaway destructured values would be something like:

```js
const [_, result] = ...
```

And then the linter won't complain. Only trouble is you can't do that if you have more than one throwaway variable. 😛